src/app/api/crypto-history/route.ts

```
@@ -0,0 +1,94 @@
+    const { searchParams } = new URL(request.url);
+    const type = searchParams.get('type') || 'latest';
+    const indicator = searchParams.get('indicator') as 'btc_dominance' | 'kimchi_premium' | 'dollar_index';
+    const hours = parseInt(searchParams.get('hours') || '24');
+    const limit = parseInt(searchParams.get('limit') || '100');
+    const offset = parseInt(searchParams.get('offset') || '0');
+
+    switch (type) {
+      case 'latest':
+        // 최신 데이터 조회 (30분 이내)
+        const latestData = await getLatestData();
+        return NextResponse.json({
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)